### DISCUSSION TITLE
Norg Guide

### DISCUSSION THREAD

**Discussion by kanodiaayush** [nvim-neorg/neorg#79] [General]
Where can I find a guide similar to [emacs org-mode basic guide](https://orgmode.org/orgguide.pdf) to learn norg (I am not coming from emacs org mode). Should I just refer this one itself and everything carries over?

**Comment by danymat** [comment]
Hmm.. 
Maybe you could read the example and the spec in `docs` folder

There will be videos explaining norg workflow as soon as 0.1 is ready to release

**Reply by kanodiaayush** [reply]
I see; thank you. Do we have a timeline for that. Also, I have been looking for things like collapsing/uncollapsing current outline for example. I could not see that in keybindings. I expect that this feature is implemented and I am just not finding it. Is this true?

**Reply by mrossinek** [reply]
Currently you can use normal vim folding for headings, lists, ranged tags and markers. See `:help folding` for the standard key bindings to open and close folds.
Folding support for more objects are likely to come in the future.

**Reply by kanodiaayush** [reply]
I see. Thank You.
